stellargraph/mapper/graphwave_generator.py

```diff
@@ -35,16 +30,18 @@ class GraphWaveGenerator:
     DataSet that contains the GraphWave embeddings.
 
     This implementation differs from the paper by removing the automatic method of calculating scales. This method was
-    found to not work well in practice, and replicating the results of the paper requires manually specifying much larger
-    scales than those automatically calculated.
+    found to not work well in practice, and replicating the results of the paper requires manually specifying much
+    larger scales than those automatically calculated.
     """
 
-    def __init__(self, G, scales=(5, 10)):
+    def __init__(self, G, scales=(5, 10), deg=20):
         """
         Args:
             G (StellarGraph): the StellarGraph object.
             scales (iterable of floats): the wavelet scales to use. Smaller values embed smaller scale structural
                 features, and larger values embed larger structural features.
+            deg: the degree of the Chebyshev polynomial to use. Higher degrees yield more accurate results but at a
+                higher computational cost. The default value of `20` is accurate enough for most applications.
```

Comment:
The Chebyshev polynomial paper I read claimed that 20 was sufficient for most applications

Reply:
And that paper was ... ?

Reply:
took me a bit too find it again! https://arxiv.org/pdf/1105.1891.pdf last paragraph of section 3

Reply:
Maybe we could note it, so that users (and us) can remember where the assertion comes from:

```suggestion
                higher computational cost. According to [1], the default value of `20` is accurate enough for most applications.
```

With a citation somewhere in this doc string:

```
[1] D. I. Shuman, P. Vandergheynst, and P. Frossard, “Chebyshev Polynomial Approximation for Distributed Signal Processing,” https://arxiv.org/abs/1105.1891
```